### PR TITLE
add Twitter + GitHub link to nav bar

### PR DESCRIPTION
**Pull request by bennn**
Suggestion for #35 

EDIT: the page doesn't look like this right now. Scroll down for the newest preview.

Here's what the nav bar looks like:
screen shot 2016-12-20 at 16 15 31

Here's what the new new members item looks like:
screen shot 2016-12-20 at 16 15 41

### CODE REVIEW

**Comment by gasche** [comment]
I think having icons is fine (it does somewhat solve @schuster's remark that it should be explicit that it takes you out of the website). I wonder if the text size isn't a bit too small compared to the icon -- for tab names they are on the small side already -- could we increase the text size a bit? I think the wording of "Follow .. on twitter" is too authoritarian, I would rephrase it as "You could follow ... on twitter" or "You may want to follow .. on twitter".

**Comment by mhyee** [comment]
I also like the icons, but they seem a little big. (Or, as @gasche says, maybe the text is too small. The text is fine on my monitor.)

Another nitpick: the icons clash a little? I wonder if minimal, single colour versions would look nicer.

> I think the wording of "Follow .. on twitter" is too authoritarian, I would rephrase it as "You could follow ... on twitter" or "You may want to follow .. on twitter".

I vaguely remember some blog post which claimed "Follow me on Twitter" got more follows than "You can follow me on Twitter." Probably because shorter is better.

How about something like "Are you on Twitter? We're @neu_prl." It's shorter, but now the "follow us" is implicit.

**Comment by bennn** [comment]
Okay here's another idea:
- Nav bar doesn't change (remove the twitter/github icons from the last commit)
- Add a "Follow" link to the bottom of the page, just like the one Frog uses.

screen shot 2016-12-20 at 17 49 47

About the wording, I picked "Follow ..." to match "Join ..." and "Sign up for ..." in the earlier items. Also I like short phrases.

**Comment by gasche** [comment]
Yes, this is very nice. (The wording is fine in these circumstances: we are not telling specific people what to do.)

**Comment by bennn** [comment]
merged! https://github.com/nuprl/website/commit/6b15415fd37b99f1a263e4ec4567730dc14fd601
